helm-chart/renku/values.yaml

```diff
@@ -45,7 +45,10 @@ global:
       value:
       overwriteOnHelmUpgrade: false
   gateway:
+    ## The client secret for the renku client application registered in keycloak.
     clientSecret: # use `uuidgen -r`
+    ## The client secret for the renku-cli client application registered in keycloak.
+    cliClientSecret: some-dummy-secret # use `uuidgen -r`
```

Comment:
Is it ok to have this dummy value in this file? Or should we add it to example configuration files?

Reply:
No, it shouldn't be here - thanks for checking.